shard.yml

```diff
@@ -20,6 +20,6 @@ development_dependencies:
   webmock:
     github: manastech/webmock.cr
 
-crystal: 0.36.1
+crystal: ">=0.36.1"
```

Comment:
```suggestion
crystal: "~> 1.0"
```

Reply:
unless you want it to be backward compatible?

Reply:
we have bumped all of our libs to be "~> 1.0"